src/main/java/org/jitsi/videobridge/rest/PublicRESTBundleActivator.java

```diff
@@ -258,6 +262,26 @@ private ServletHolder initializeProxyServlet(
                     holder.setInitParameter("hostHeader", hostHeader);
 
                 servletContextHandler.addServlet(holder, pathSpec);
+
+                // CORS
+                String allowedOrigins
+                    = ConfigUtils.getString(
+                        cfg,
+                        JETTY_PROPERTY_PREFIX
+                            + JETTY_CORS_ALLOWED_ORIGINS,
+                        null);
+                if (allowedOrigins != null && allowedOrigins.length() != 0)
+                {
+                    FilterHolder filterHolder = servletContextHandler.addFilter(
+                        org.eclipse.jetty.servlets.CrossOriginFilter.class,
+                        "/*",
+                        EnumSet.of(DispatcherType.REQUEST)
+                    );
+                    filterHolder.setInitParameter(
+                        "allowedOrigins",
```

Comment:
Nits: can you please use the `CrossOriginFilter.ALLOWED_ORIGINS_PARAM` constant, and use the simple class name above (no need to use the full name, since the package is imported)